demisto_sdk/commands/common/hook_validations/release_notes.py

```diff
@@ -15,19 +16,50 @@ class ReleaseNotesValidator:
         master_diff (str): the changes in the changelog file compared to origin/master.
     """
 
-    def __init__(self, file_path):
+    def __init__(self, file_path, modified_files=None, pack_name=None, added_files=None):
         self.file_path = file_path
+        self.modified_files = modified_files
+        self.added_files = added_files
+        self.pack_name = pack_name
         self.release_notes_path = get_release_notes_file_path(self.file_path)
         self.latest_release_notes = get_latest_release_notes_text(self.release_notes_path)
 
+    def are_release_notes_complete(self):
+        is_valid = True
+        if self.modified_files:
+            for file in self.modified_files:
+                if 'README' in file:
+                    continue
+                elif self.pack_name in file:
```

Comment:
try to make this code more self explanatory. this is fore the lines following this comment.